addons/crm/tests/test_crm_lead_assignment.py

```diff
@@ -533,3 +535,65 @@ def test_no_assign_if_exceed_max_assign(self):
         members_data = sales_team_4._assign_and_convert_leads()
         self.assertFalse(members_data,
             "If team member has lead count greater than max assign,then do not assign any more")
+
+    def test_assign_preferred_domain(self):
+        """ Test preferred domain use """
+
+        random.seed(1914)
+
+        preferred_tag = self.env['crm.tag'].create({'name': 'preferred'})
+
+        leads = self._create_leads_batch(
+            lead_type='lead',
+            user_ids=[False],
+            count=11,
+        )
+        leads[:8].write({'tag_ids': [(6, 0, preferred_tag.ids)]})
+        # commit probability and related fields
+        leads.flush_recordset()
+        self.assertInitialData()
+        sales_team_5 = self.env['crm.team'].create({
```

Comment:
We don't really care about the numbering, do we ? It is just specific to this test. And why not using an existing data ?
```suggestion
        test_sales_team = self.env['crm.team'].create({
```